components/plurals/data/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+# This file is part of ICU4X. For terms of use, please see the file
```

Comment:
oh I see, you implemented separate crates but distributed them inside the individual components instead of in a centralized location

Reply:
Let's please not block on where in the repo these will live